lib/strava/models/mixins/http_response.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Strava
+  module Models
+    module Mixins
+      module HttpResponse
+        extend ActiveSupport::Concern
+
+        attr_accessor :http_response
+
+        def ratelimit
```

Comment:
Do we want this very much? It's unlikely that Strava adds an `http_response` property to models, but `ratelimit` is more likely :) I don't mind personally always having to type `.http_response.ratelimit`.